### ISSUE TITLE
SelectOneRadio: items are not selected when moving the focus with arrow keys

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#9088]
### Describe the bug

Items are no longer selected when moving the focus with arrow keys. They are in 11.

### Reproducer

https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml

### Expected behavior

Items should be selected when moving the focus with arrow keys.

### PrimeFaces version

12

**Comment by jepsar** [comment]
@mertsincan FYI I'm getting "Uncaught TypeError: Cannot read properties of undefined (reading 'keyCode')" (now(?)).

<img width="380" alt="Screenshot 2022-09-03 at 08 46 01" src="https://user-images.githubusercontent.com/7500178/188259268-b637a0b0-3987-4e88-8e26-9b6ffabc1871.png">

I'm currently working on a fix for #9137 and will have a look if I can fix this while I'm at it.

**Comment by jepsar** [comment]
This resource was missing:
```java
@ResourceDependency(library = "primefaces", name = "jquery/jquery-plugins.js")
```
